packages/init/src/index.ts

```diff
@@ -19,7 +20,14 @@ async function init() {
   const lib: string | undefined = findArgument(process.argv, "library");
   const branch: string | undefined = findArgument(process.argv, "branch");
   const isTrackingAvailable =
-    findArgument(process.argv, "tracking") === "false" ? false : true;
+    findArgument(process.argv, "tracking") !== "false";
+
+  Tracker.get().initialize(
```

Comment:
Does the tracker.get still make sense here ?
Just a challenge, we don't need to modify it but i'm wondering since it's a script does it make sense to secure the tracker and all knowing that we could instantiate it in another file and just call the helpers here.
Like instantiate it in memory and then see if we want to call the helpers or not